konduit-serving-api/src/test/java/ai/konduit/serving/data/GeneratedClassesTest.java

```diff
@@ -0,0 +1,47 @@
+package ai.konduit.serving.data;
```

Comment:
Move all classes/tests added in this PR from konduit-serving-api to konduit-serving-pipeline - everything should be in that module, no additions to konduit-serving-api

Reply:
Moved. This file is to be removed - just used it to verify successful code generation. Looks like it doesn't make sense to test this API, covered by higher level tests anyway.